test/__init__.py

```diff
@@ -45,14 +45,14 @@
 import pymongo
 import pymongo.errors
 from bson.son import SON
-from pymongo import common, message
-from pymongo.common import partition_node
-from pymongo.database import Database
 from pymongo.hello import HelloCompat
-from pymongo.mongo_client import MongoClient
 from pymongo.server_api import ServerApi
 from pymongo.ssl_support import HAVE_SSL, _ssl  # type:ignore[attr-defined]
-from pymongo.uri_parser import parse_uri
+from pymongo.synchronous import common, message
+from pymongo.synchronous.common import partition_node
+from pymongo.synchronous.database import Database
+from pymongo.synchronous.mongo_client import MongoClient
+from pymongo.synchronous.uri_parser import parse_uri
```

Comment:
I think we should revert all the test import changes because we want to be sure the old imports still work.

Reply:
We have a lot of tests that import private modules, so we'd either have to re-export everything at the top level or have a mix of `pymongo.module` and `pymongo.synchronous.module` imports in the tests.

Reply:
Right, anything private is free to change but the public imports should remain the same.

Reply:
Of note, we will most likely be using unasync for the tests as well, so we'll need to have the qualified imports.

Reply:
Ah good point! Relying on test_import_compatibility SGTM.